roles/ssp/defaults/main.yml

```diff
@@ -2,11 +2,14 @@
 #
 ---
 
-ssp_version: "1.17"
+ssp_version: "2.1.0"
 
-ssp_repo_url: "git://github.com/simplesamlphp/simplesamlphp.git"
+ssp_repo_url: "https://github.com/simplesamlphp/simplesamlphp.git"
 ssp_repo_version: "simplesamlphp-{{ ssp_version }}"
-ssp_path: "/var/simplesamlphp"
+# Uncomment in order to install using the release
+# ssp_release_url: "https://github.com/simplesamlphp/simplesamlphp/releases/download/v{{ ssp_version }}/{{ ssp_repo_version }}-full.tar.gz"
+ssp_basepath: "/var/simplesamlphp"
+ssp_path: "{{ ssp_basepath }}/{{ ssp_baseurlpath }}-{{ ssp_version }}"
```

Comment:
There is no need to change the default ("/var/simplesamlphp"). It's better to override the `ssp_path` in the inventory.
Note that `ssp_path` is changed with set_fact *only* if `ssp_version_suffix` is defined in the inventory